suite-native/module-settings/src/components/CurrencySelector.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { Select, SelectValue } from '@suite-native/atoms';
+import { CurrencyType, fiatCurrenciesMap } from '@suite-common/suite-config';
+
+import { selectCurrency, setCurrency } from '../slice';
+import { transformFiatCurrencyToSelectItem } from '../utils';
+
+export const CurrencySelector = () => {
```

Comment:
There is back arrow visible in the header of opened BottomSheet, please hide it because cross at right corder enough. Back arrow is meant to be used in multistep BottomSheets.

Reply:
https://github.com/trezor/trezor-suite/pull/5895/commits/088ccfb38beef6c123b6dd7a50d6ae4c2f120a07